lighthouse-plots/measure.js

```diff
@@ -0,0 +1,185 @@
+  // Also removed sites that don't have significant index pages:
+  // "t.co", "popads.net", "onclickads.net", "microsoftonline.com", "onclckds.com", "cnzz.com",
+  // "live.com", "adf.ly", "googleusercontent.com",
+  'https://google.com',
+  'https://youtube.com',
+  'https://facebook.com',
+  'https://baidu.com',
+  'https://wikipedia.org',
+  'https://yahoo.com',
+  'https://amazon.com',
+  'http://www.qq.com/',
+  'https://taobao.com',
+  'https://vk.com',
+  'https://twitter.com',
+  'https://instagram.com',
+  'http://www.hao123.cn/',
+  'http://www.sohu.com/',
+  'https://sina.com.cn',
+  'https://reddit.com',
+  'https://linkedin.com',
+  'https://tmall.com',
+  'https://weibo.com',
+  'https://360.cn',
+  'https://yandex.ru',
+  'https://ebay.com',
+  'https://bing.com',
+  'https://msn.com',
+  'https://www.sogou.com/',
+  'https://wordpress.com',
+  'https://microsoft.com',
+  'https://tumblr.com',
+  'https://aliexpress.com',
+  'https://blogspot.com',
+  'https://netflix.com',
+  'https://ok.ru',
+  'https://stackoverflow.com',
+  'https://imgur.com',
+  'https://apple.com',
+  'http://www.naver.com/',
+  'https://mail.ru',
+  'http://www.imdb.com/',
+  'https://office.com',
+  'https://github.com',
+  'https://pinterest.com',
+  'https://paypal.com',
+  'http://www.tianya.cn/',
+  'https://diply.com',
+  'https://twitch.tv',
+  'https://adobe.com',
+  'https://wikia.com',
+  'https://coccoc.com',
+  'https://so.com',
+  'https://fc2.com',
+  'https://www.pixnet.net/',
+  'https://dropbox.com',
+  'https://zhihu.com',
+  'https://whatsapp.com',
+  'https://alibaba.com',
+  'https://ask.com',
+  'https://bbc.com'
+];
+
+async function main() {
+  if (utils.isDir(OUT_PATH)) {
+    console.log('Please remove or move ./out/');
+    return;
+  }
+
+  let launcher;
+  try {
+    launcher = new ChromeLauncher({ port: 9222, autoSelectChrome: true });
+    try {
+      await launcher.isDebuggerReady();
+    } catch (err) {
+      await launcher.run();
+    }
+    await runAnalysis();
+    await launcher.kill();
+  } catch (err) {
+    await launcher.kill();
+    console.log('err', err);
+  }
+}
+
+main();
+
+async function runAnalysis() {
+  // Running it n + 1 times because the first run might not be recorded
+  for (let i = 0; i <= NUMBER_OF_RUNS; i++) {
+    const id = i.toString();
+    // Might not want to record the first run because it has different perf characteristics from
+    // subsequent runs (e.g. DNS cache which can't be easily reset between runs)
+    const isFirstRun = false;
+
+    for (let url of URLS) {
+      console.log('Measuring site:', url, 'run:', i);
+      await singleRunAnalysis(url, id, isFirstRun);
+    }
+  }
+}
+
+/**
+ * @param {string} url
+ * @param {string} id
+ * @param {boolean} ignoreRun
+ */
+async function singleRunAnalysis(url, id, ignoreRun) {
+  const parsedURL = parseURL(url);
+  const host = `${parsedURL.host}-${parsedURL.pathname}`
```

Comment:
can we call this `filenameHost` or something?


Also I wrote this bit for traceviewer in case you want to reuse:
```js
      var illegalRe = /[\/\?<>\\:\*\|":]/g;
      var controlRe = /[\x00-\x1f\x80-\x9f]/g;
      var reservedRe = /^\.+$/;

return filename
          .replace(illegalRe, '.')
          .replace(controlRe, '\u2022')
          .replace(reservedRe, '')
          .replace(/\s+/g, '_');
```